CMakeLists.txt

```diff
@@ -75,7 +75,7 @@ include_directories(${ROCM_OCL_INCLUDES})
 endif()
 
 if(UNIX)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -fno-rtti -Wall")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -fno-rtti -Wall")
```

Comment:
LGTM, but please make `-std=c++14` and `-Wall` platform independent.

Reply:
Thanks, I will fix in a follow up change. (need to get this in ASAP).

Reply:
Btw, for MSVC in option `std` a colon should be instead of an equals sign: `/std:c++14`, otherwise warning `ignoring unknown option`.